plugins/hydra_ray_launcher/example/conf/mode/aws.yaml

```diff
@@ -0,0 +1,35 @@
+hydra:
```

Comment:
Please rebase and update your configs, including a @package.
we can talk about it when we meet.